### PR TITLE
Use erblint to maintain consistency within ERBs

### PR DESCRIPTION
**Pull request by mjgiarlo**
Supersedes #2867 (HT: @justinlittman)

## Why was this change made? 🤔

Extend our linting & style-checking to ERB files, of which H2 has a great many.

Includes:
* Add erblint dependency for linting ERB files (& supporting rake task)
* Run erblint as part of CI
* Autocorrect erblint violations

## How was this change tested? 🤨

CI

### CODE REVIEW

File: .erb-lint.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+exclude:
+  - '**/vendor/**/*'
```
Comment (mjgiarlo) [inline .erb-lint.yml:3]: Without this exclusion, the erblint build in CI fails due to the existence of Ruby files in `vendor/bundle/`.

File: .erb-lint.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      Layout/LineLength:
+        Enabled: false
+      Layout/TrailingEmptyLines:
+        Enabled: false
+      Style/FrozenStringLiteralComment:
+        Enabled: false
+      Lint/UselessAssignment:
+        Enabled: false
+      Rails/OutputSafety: # Already covered by ErbSafety linter
+        Enabled: false
```
Comment (mjgiarlo) [inline .erb-lint.yml:22]: I tried removing these but they muck up the autocorrect mechanism. There's a reason the erblint README recommends keeping them off.

File: lib/tasks/linters.rake [hunk 3]
```diff
@@ -3,12 +3,20 @@
...
 rescue LoadError
   task rubocop: :environment do
     abort "Please install the rubocop gem to run rubocop."
   end
 end
+
+task :erblint do
+  puts "Running erblint..."
+  `erblint --lint-all --format compact`
+end
```
Comment (mjgiarlo) [inline lib/tasks/linters.rake:22]: The erblint gem doesn't ship a rake task (intentionally), so we patch in our own. This is recommended in a related erblint PR.

File: lib/tasks/linters.rake [hunk 4]
```diff
@@ -3,12 +3,20 @@
 begin
   require "rubocop/rake_task"
   RuboCop::RakeTask.new do |task|
+    task.requires << "standard"
     task.requires << "rubocop-performance"
     task.requires << "rubocop-rails"
     task.requires << "rubocop-rspec"
+    task.requires << "rubocop-capybara"
+    task.requires << "rubocop-factory_bot"
```
Comment (mjgiarlo) [inline lib/tasks/linters.rake:11]: Missed these in the mega-standardrb PR.

**Review by justinlittman — APPROVED** [review]
Add something to README or Dev Playbook about how to run the lint?

**Comment by mjgiarlo** [comment]
@justinlittman 💬 
> Add something to README or Dev Playbook about how to run the lint?

Great idea: [`48b32a9` (#3115)](https://github.com/sul-dlss/happy-heron/pull/3115/commits/48b32a92d338c2cadd86532e9d06b42197b9d5f9)

**Pull request merged**